private-templates-service/adaptivecards-templating-service/src/models/models.ts

```diff
@@ -37,10 +50,16 @@ export interface JSONResponse<T> {
   result?: T;
 }
 
+export enum TemplateState {
+  draft = "draft",
```

Comment:
Yes, because **TemplateState = 0 or 1 or 2** doesn't really say what it is and front end will have to match the enum to be able to differentiate between the states.